micromamba/tests/test_create.py

```diff
@@ -1441,3 +1443,55 @@ def test_create_empty_or_absent_dependencies(tmp_path):
         "-p", env_prefix, "-f", tmp_path / "env_spec_absent_dependencies.yaml", "--json"
     )
     assert res["success"]
+
+
+env_spec_empty_lines_and_comments = """
+# The line below are empty (various number of spaces)
+"""
+
+env_spec_empty_lines_and_comments += "\n"
+env_spec_empty_lines_and_comments += "  \n"
+env_spec_empty_lines_and_comments += "    \n"
+env_spec_empty_lines_and_comments += "	\n"
+env_spec_empty_lines_and_comments += "# One comment \n"
+env_spec_empty_lines_and_comments += "	@ yet another one with a prefixed by a tab\n"
+env_spec_empty_lines_and_comments += "wheel\n"
+
+env_repro_1 = """
+wheel
+
+setuptools
+"""
+
+env_repro_2 = """
+wheel
+setuptools
+
+# comment
+"""
+
+
+@pytest.mark.parametrize("env_spec", [env_spec_empty_lines_and_comments, env_repro_1, env_repro_2])
+def test_create_with_empty_lines_and_comments(tmp_path, env_spec):
+    # Non-regression test for:
+    #  - https://github.com/mamba-org/mamba/issues/3289
+    #  - https://github.com/mamba-org/mamba/issues/3659
+    memory_limit = 100  # in MB
```

Comment:
@jjerphan could we consider increasing to 150? It does not work consistently on my machine (~116mB).